instance/management/commands/activity_csv.py

```diff
@@ -70,13 +70,19 @@ def handle(self, *args, **options):
 
         self.activity_csv(out)
 
-    def activity_csv(self, out):  # pylint: disable=missing-docstring
-        # Produce a mapping of public IPs (of active app servers) to parent instances.
-        active_appservers = {
-            instance.active_appserver.server.public_ip: instance for instance in OpenEdXInstance.objects.all()
-            if not (instance.active_appserver is None or instance.active_appserver.server.public_ip is None)
-        }
-
+    @staticmethod
+    def get_active_appservers():
+        """ Produce a mapping of public IPs (of active app servers) to parent instances. """
+        active_appservers = {}
+        for appserver in OpenEdXAppServer.objects.filter(_is_active=True):
+            public_ip = appserver.server.public_ip
+            if public_ip is not None:
+                active_appservers[public_ip] = appserver.instance
+        return active_appservers
+
+    def activity_csv(self, out):
+        """Generate the activity CSV."""
+        active_appservers = self.get_active_appservers()
```

Comment:
@antoviaque We do output the "Internal LMS Domain", which if you sort on that column, will group your appservers.

Alternately, I can sort the `active_appserver` list by `instance.id` when [iterating through the appserver `public_ip` address](https://github.com/open-craft/opencraft/pull/181/files/d27df1bfc8b646eaf47644d7b8af9bc6b467257f#diff-827bdb29a85e840ec5396164d71be8ddR128).